test/system/package-managers/rpm.spec.ts

```diff
@@ -18,4 +19,13 @@ describe("rpm package manager tests", () => {
 
     expect(pluginResult).toMatchSnapshot();
   });
+
+  it("should correctly analyze an rpm image with sqlite DB", async () => {
+    const image = "amazonlinux:2022.0.20220504.1";
+    const pluginResult = await scan({
+      path: image,
+    });
+
+    expect(pluginResult).toMatchSnapshot();
+  });
```

Comment:
I see that the tests we have for RPM (actually for all package managers) are quite pretty short. Are these the only testing we do? Do we want to have more test cases? There might not be interesting test cases, just asking.
For example, do we want to test that we can handle the case of a broken sqlite file? Or this not a likely thing to encounter?

Reply:
We have a couple of more tests related to the data in `rpm-parser`.
I think it's extremely unlikely - if the file is corrupted than the OS will be broken I believe